### PR TITLE
Use django rest framework api views for generic APIs

### PR DESCRIPTION
**Pull request by lydiagu**
@marcua @jrbotros what do we think about using rest framework to reduce work for CRUD apis?

### CODE REVIEW

File: example_project/example_project/orchestra_settings.py [hunk 1]
```diff
@@ -108,7 +108,10 @@ def setup_orchestra(settings_module_name):
 
     # Don't authenticate users without a view explicitly calling for it
     settings.REST_FRAMEWORK = {
-        'DEFAULT_AUTHENTICATION_CLASSES': (),
+        'DEFAULT_AUTHENTICATION_CLASSES': (
```
Comment (marcua) [inline example_project/example_project/orchestra_settings.py:110]: remind me to think about this

Comment (lydiagu) [inline example_project/example_project/orchestra_settings.py:110]: reminding

I can see us removing BasicAuth - we don't use it and probably shouldn't allow it

**Comment by marcua** [comment]
LGTM assuming the Orchestra API for creating projects and the user interface still loads.

**Comment by lydiagu** [comment]
yup can create projects and user interface is good.

**Pull request merged**